services/src/main/resources/config-spring-geonetwork.xml

```diff
@@ -76,21 +77,28 @@
             <value>Conformance Class 2: INSPIRE data sets and data set series interoperability metadata.</value>
           </array>
         </entry>
-        <entry key="TG version 2.0 - Spatial data service">
+        <entry key="TG version 2.0 - Network services">
           <array>
             <value>Common Requirements for ISO/TC 19139:2007 based INSPIRE metadata records.</value>
-            <value>Conformance Class 1: INSPIRE data sets and data set series baseline metadata.</value>
-            <value>Conformance Class 2: INSPIRE data sets and data set series interoperability metadata.</value>
+            <!--<value>Conformance Class 1: INSPIRE data sets and data set series baseline metadata.</value>
+            <value>Conformance Class 2: INSPIRE data sets and data set series interoperability metadata.</value>-->
             <value>Conformance Class 3: INSPIRE Spatial Data Service baseline metadata.</value>
-            <!--<value>Conformance Class 4: INSPIRE Network Services metadata.</value>
-            <value>Conformance Class 5: INSPIRE Invocable Spatial Data Services metadata.</value>
+            <value>Conformance Class 4: INSPIRE Network Services metadata.</value>
+            <!--<value>Conformance Class 5: INSPIRE Invocable Spatial Data Services metadata.</value>
             <value>Conformance Class 6: INSPIRE Interoperable Spatial Data Services metadata.</value>
             <value>Conformance Class 7: INSPIRE Harmonised Spatial Data Services metadata.</value>-->
           </array>
         </entry>
       </map>
     </property>
+    <property name="testsuitesConditions">
+      <map>
+        <entry key="TG version 2.0 - Data sets and series" value="gmd:hierarchyLevel[*/@codeListValue='dataset']" />
```

Comment:
"Data sets and series"

Reply:
The value is the same that was already in `testsuites` section from the `TG version 2.0 - Data sets and series`. I´ll update in both places to remove the space in `data sets`.

Reply:
We need to change the xpath to `gmd:hierarchyLevel[*/@codeListValue = ('dataset', 'series')]`

Reply:
Thanks for the clarification, was not sure if the problem was the test name.